certbot-apache/local-oldest-requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Remember to update setup.py to match the package versions below.
-acme[dev]==0.29.0
-certbot[dev]==1.6.0
\ No newline at end of file
+acme[dev]==1.8.0
+certbot[dev]==1.10.0
```

Comment:
This should be 11 now

Reply:
1.10.0 is when the `notify` function was added. Is there a reason to require a newer version?

Reply:
Ah, nope, I actually assumed you meant to set it to the current version, which is not the case.